google/datalab/bigquery/_table.py

```diff
@@ -103,8 +103,11 @@ class Table(object):
   # Allowed characters in a BigQuery table column name
   _VALID_COLUMN_NAME_CHARACTERS = '_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
 
-  # When fetching table contents, the max number of rows to fetch per HTTP request
+  # When fetching table contents for a range or iteration, use a small page size per request
```

Comment:
So based on that data, should this constant be `100` instead of `1024`?

(Again, probably worth splitting off into a different issue, since it's largely orthogonal.)